openfl-tutorials/interactive_api/jax_linear_regression/envoy/envoy_config_2.yaml

```diff
@@ -0,0 +1,13 @@
+params:
+  cuda_devices: []
+
+optional_plugin_components: {}
+
+shard_descriptor:
+  template: linreg_shard_descriptor.LinRegSD
+  params:
+    rank: 43 # Dataset seed
```

Comment:
@MasterSkepticista  Not sure why you think this should hold only federation related config only. From the examples I see the SD related configs are defined here. The shard descriptor is created from here, so parameterizing is not possible. Am I missing something?

Reply:
This is the only way of parameterizing the `ShardDescriptor`.
Args like `rank`, `img_size`, `transform`, or `data_path` can make it to the `params`, because they're relevant to the federation tasks of collaborators in general. Bottom 3 args in this config are for generating any random data and should be better done in the experiment instead of controlling via `envoy_config`